### ISSUE TITLE
Release MMCLIP

### ISSUE THREAD

**Issue by milkyfun0** [AIGeeksGroup/MMCLIP#2]
First of all, I sincerely thank you and your team for their efforts in [MMCLIP: Cross-modal Attention Masked Modelling for Medical Language-Image Pre-Training
Outstanding work in the paper! We have noticed that the code has not been made public yet, so we would like to venture to ask if there are any plans to open the source code?
